portfolio/src/main/java/com/google/sps/comment/Comment.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.comment;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+
+import java.io.IOException;
+
+public class Comment {
+    private String text;
+    private long timestamp;
+    private int score;
+
+    public Comment(String message) throws IOException {
+        this.text = message;
+        this.timestamp = System.currentTimeMillis();
+        setSentimentScore(message);
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public long getTime() {
+        return timestamp;
+    }
+
+    public int getSentimentScore() {
+        return score;
+    }
+
+    private void setSentimentScore(String message) throws IOException {
```

Comment:
you can access the message from this.text, there is no need to make it a parameter.
I also feel that "initializeSentimentScore" is a better function name, since it is only called at the constructor, and the function is private.

Reply:
I've made the change